examples/deployment/docker-compose.yml

```diff
@@ -3,37 +3,54 @@ services:
   mysql:
     image: mariadb:10.1
     environment:
-      - MYSQL_ROOT_PASSWORD=$DB_PASSWORD
+      - MYSQL_ROOT_PASSWORD
   trillian-db-seed:
     build:
       context: ../..
       dockerfile: ./examples/deployment/docker/db_client/Dockerfile
     environment:
-      - DB_USER=root
-      - DB_PASSWORD=$DB_PASSWORD
-    command: ./wait-for-it.sh -t 0 mysql:3306 -- ./scripts/resetdb.sh --verbose --force -h mysql
+      - MYSQL_ROOT_USER=root
+      - MYSQL_ROOT_PASSWORD
+      - MYSQL_HOST=mysql
+      - MYSQL_DATABASE=test
+      - MYSQL_USER=test
+      - MYSQL_PASSWORD=zaphod
+      - MYSQL_USER_HOST=%
+    command: ./wait-for-it.sh -t 0 mysql:3306 -- ./scripts/resetdb.sh --verbose --force
   trillian-log-server:
     build:
       context: ../..
       dockerfile: examples/deployment/docker/log_server/Dockerfile
+    entrypoint: [
+      "/trillian_log_server",
+      "--storage_system=mysql",
+      "--mysql_uri=test:zaphod@tcp(mysql:3306)/test",
+      "--rpc_endpoint=0.0.0.0:8090",
+      "--http_endpoint=0.0.0.0:8091",
+      "--alsologtostderr",
+    ]
     restart: always # retry while mysql is starting up
     ports:
       - "8090:8090"
       - "8091:8091"
     depends_on:
-      - mysql
-    environment:
-      - DB_USER=root
-      - DB_PASSWORD=$DB_PASSWORD
+      - trillian-db-seed
```

Comment:
Is `depends_on` correct?  
docker compose doesn't wait for the dependency to be healthy, only started.  Does that make a difference?